script/foreman-debug

```diff
@@ -19,6 +19,8 @@ This program can be used on Foreman instances, Smart Proxy instances or
 backend services separately.
 
 OPTIONS:
+  -E      Detailed output incl. host names or domains
+          (THIS OPTION ADDS MORE SENSITIVE DATA IN THE REPORT)
```

Comment:
What size difference is there between the selective tables and dumping the database?  You've listed some potentially large tables anyway, there don't seem to be many in my opinion that would be any larger than say, `hosts` or `nics`.